src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java

```diff
@@ -21,31 +18,37 @@
 
 public class MessageWall extends WikiBasePageObject {
 
-  final By firstMessageWrapperBy =
+  private static final By focusedNewMessageFormBy = By.cssSelector("#wall-new-message.focused");
+
+  private final static By firstMessageWrapperBy =
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)